### PR TITLE
feat(console): platform label in connectors table

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Implement platform labels in connectors table.

## Linear Issue Reference

LOG-2819

## Testing

截屏2022-06-02 下午2 48 28

### CODE REVIEW

**Review by gao-sun — COMMENTED** [review]
is it too tight between connector name and platform row?

**Comment by wangsijie** [comment]
Add padding between name and platform row.

image

File: packages/console/src/pages/Connectors/components/ConnectorName/index.tsx [hunk 1]
```diff
@@ -41,7 +45,24 @@ const ConnectorName = ({ type, connector, onClickSetup, isShowId = false }: Prop
...
+                {connectors.map(
+                  ({ id, platform }) =>
+                    platform && (
+                      <div key={id} className={styles.platform}>
+                        <ConnectorPlatformIcon platform={platform} />
+                        {t(connectorPlatformLabel[platform])}
+                      </div>
+                    )
+                )}
+              </div>
```
Comment (simeng-li) [inline packages/console/src/pages/Connectors/components/ConnectorName/index.tsx:62]: just wondering should those be  wrapped up in `type === ConnectorType.Social`?

**Review by simeng-li — APPROVED** [review]
over all LGTM

File: packages/console/src/pages/Connectors/components/ConnectorName/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/ConnectorName/index.tsx]: You are right, that's more accurate, updated

**Pull request merged**